src/views/workflow-history/workflow-history-timeline-group/workflow-history-timeline-group.styles.ts

```diff
@@ -24,12 +24,24 @@ const cssStylesObj = {
     display: 'flex',
     flexDirection: 'column',
   },
-  timelineEventHeader: {
+  timelineEventHeader: (theme) => ({
     display: 'flex',
     alignItems: 'center',
-    gap: '16px',
-    padding: '12px 0',
-  },
+    gap: theme.sizing.scale600,
+    padding: `${theme.sizing.scale500} 0`,
```

Comment:
nit: just double-check that this shorthand property does not conflict with any longhand properties.

Reply:
It won't since we are using monolithic engine & we are not mixing it with any other padding props in the same style or overriding other styled components.

Docs ref:
https://styletron.org/concepts#shorthand-and-longhand-properties